### DISCUSSION TITLE
Is it possible to disable the mini preview while scrolling through in the big previewer?

### DISCUSSION THREAD

**Discussion by Joeyndot** [QL-Win/QuickLook#1211] [Q&A] [answered]
Basically if you hover on the file, itll show a s=mini preview but if you hit space it populates the big previewer. But if you use the arrow keys, the mini previewer will overlay on top of the big one and impedes the view. Just wondering if theres a way to fix that.

**Comment by rabelux** [comment]
QuickLook doesn't provide a mini preview. That must be some other tool.

**Reply by Joeyndot** [reply]
So this is another program? I'm not sure what it could be... tough ask, but any ideas? lol 

![image](https://user-images.githubusercontent.com/117954012/201277700-4d07834a-aa27-4a6d-b2e7-b8f2dbfb517a.png)
![image](https://user-images.githubusercontent.com/117954012/201277459-9988418b-3e7e-4a5b-9637-c92a22dfa824.png)

**Comment by xupefei** [comment, marked as answer]
Looks like http://qttabbar.wdfiles.com/local--files/documents/preview.html.

**Reply by Joeyndot** [reply]
love u
